src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setColor(colorSupplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
1) Move the plain number to constant.
2) random.nextInt(100) will never return 100. Look up nextInt().